### DISCUSSION TITLE
Agentless bitwarden client

### DISCUSSION THREAD

**Discussion by sewnie** [quexten/goldwarden#158] [Ideas]
Hi, i would like to make a tiny bitwarden client in Go that serves as a agentless method for retrieving passwords, upon user giving the password. I do not want an agent running on my system.

Could it be possible that the `agent/bitwarden` package be exported to a new separate package for other go programs to use? Thanks.

**Comment by sewnie** [comment]
The alternative workaround for me seems to encrypt a bitwarden vault export with age, and make a tiny shell script to get keys.

**Comment by quexten** [comment]
Created a feature request issue for this. Sounds interesting, though I haven not considered the implications of this. I think quite a bit of the agent code expects to run within the context of goldwarden. Might need a rather large refactor (although that might be needed either way).

**Comment by sewnie** [comment]
What is the status so far?

**Reply by fionnb** [reply]
It looks like goldwarden development has pretty much come to a halt for months. It seems not even pull requests are evaluated anymore.
